xgboost_ray/sklearn.py

```diff
@@ -557,6 +561,8 @@ def get_num_boosting_rounds(self):
 
 RayXGBRFRegressor.__doc__ = _treat_estimator_doc(XGBRFRegressor.__doc__)
 
+PublicAPI(stability="beta")
```

Comment:
Should this be here?

Reply:
Ah that happened a few times, will fix in a sec...